packages/svelte/src/internal/client/error-handling.js

```diff
@@ -72,9 +77,18 @@ function adjust_error(error, effect) {
 		context = context.p;
 	}
 
-	define_property(error, 'message', {
-		value: error.message + `\n${component_stack}\n`
-	});
+	if (!message_descriptor || message_descriptor.configurable) {
+		define_property(error, 'message', {
+			value: error.message + `\n${component_stack}\n`
+		});
+	} else {
+		// eslint-disable-next-line no-console
+		console.error(
+			"The following it's not a separate error: we usually modify the error message to show you this information but `message` was non configurable so we print them in a separate log.\n" +
+				error.message +
+				`\n${component_stack}\n`
+		);
```

Comment:
This is ... a lot 😄 I would trim this down to just this, or just not log anything at all. I don't know how many people will have worse stacks as a result, but I doubt it's many
```suggestion
		console.error(
			"[adjusted stack] " +
				error.message +
				`\n${component_stack}\n`
		);
```